src/styles/utils/variables.scss

```
@@ -0,0 +1,15 @@
+$clock-size: 300px;
+$circle: 50%;
+$minutes-arrow-height: 50%;
+$minutes-arrow-width: 2%;
+$minutes-arrow-color: green;
+$base-pos-value: 50%;
+$center-hight: 8%;
+$center-width: 8%;
+$center-color: orange;
```

This is more readable, when you read css without looking into variables.scss file
```suggestion
$color-orange: orange;
```